### ISSUE TITLE
test_delete_cookie fails with Python 3.5

### ISSUE THREAD

**Issue by felixonmars** [bottlepy/bottle#791]
Hi, I am getting the following test error when running against Python 3.5:

```
======================================================================
FAIL: test_delete_cookie (test_environ.TestResponse)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/python-bottle/src/bottle/test/test_environ.py", line 614, in test_delete_cookie
    self.assertTrue('name=;' in cookies[0])
AssertionError: False is not true

----------------------------------------------------------------------
```

With Python 3.4 and all other components at the same version, it passes.

**Comment by kingFighter** [comment]
Confirm with `python3.5` under `win8`.
BTW, @felixonmars Can you tell me how to setup `bottle` test environment. 
I couldn't find it in `google`, `http://bottlepy.org/` and  `google groups`

**Comment by oz123** [comment]
@defnull , I know you are not so keen on adding another CI, but https://ci.appveyor.com offers CI for windows. Such issues can than be caught, without having to work directly with Windows (I don't own a Windows Laptop either ...).

**Comment by defnull** [comment]
Cookie test is fixed now in `release-0.12` branch (will be part of next release, whenever that happens).
